cmd/kib.go

```diff
@@ -58,7 +59,7 @@ func processInt(st string) string {
 }
 
 // ExtractData extract data to card
-func ExtractData(mainHTML *goquery.Selection) Card {
+func ExtractData(config siteConfig, mainHTML *goquery.Selection) Card {
```

Comment:
What do you think if we use different `ExtracData` for each language ? It would be specify in the `siteConfig`. Or maybe add language specific keyword in `sideConfig`

Reply:
There would be a lot of duplicated code. I don't think there's a huge benefit to doing so right now. If the two languages diverge significantly, and thus the code would be less duplicated, then I think it would make more sense.

Reply:
I'm not fond of all those "or" we have a `config` object we should use it. It might be a lot a duplicated code, but I think a more straightforward code is better.